Cappua/cappua-fe
# Setup

#### What does  this PR do?
- add featured artisit data 
- Remove conditional rendering 
- adjust styling for Trio
- Setup for track pages 
- add soundbard and track nav to track pages 
- add track title and formatting 
#### Where should the reviewer start?
- TrackNav.js 
- TrackCard.js
#### How should this be manually tested?
- test by running `npm start` in the terminal and check for load without errors and bugs
